docs/sources/operators-guide/configuring/configuring-shuffle-sharding/index.md

```diff
@@ -194,6 +195,12 @@ When you enable compactor shuffle sharding by setting `-compactor.compactor-tena
 
 You can override the compactor shard size on a per-tenant basis setting by `compactor_tenant_shard_size` in the overrides section of the runtime configuration.
 
+### Alertmanager shuffle sharding
+
+Alertmanager only distributes across replicas by tenant, the state and workload it is not divided any further. The replication factor setting `-alertmanager.sharding-ring.replication-factor` dictates how many replicas are used for a tenant.
+
+As a consequence, shuffle sharding is effectively always enabled for Alertmanager.
```

Comment:
```suggestion
As a result, shuffle sharding is effectively always enabled for Alertmanager.
```
What does "effectively" mean here? Is it or is it not enabled?

Reply:
It means that there is no explicit method to enable "shuffle sharding", because the effect of limiting tenants to a fixed number of replicas is inherent in the design.